install/kubernetes/cilium/values.yaml

```diff
@@ -668,7 +668,7 @@ hubble:
     #
     enabled: ~
     # -- Configure the port the hubble metric server listens on.
-    port: 9091
+    port: 9965
```

Comment:
After #20066 was merged, now these changes need to be made to `install/kubernetes/cilium/values.yaml.tmpl` instead, then run `make -C install/kubernetes` to update this file.

Reply:
I did a rebase and changed the `values.yaml.tmpl` file but I get this error:

```
make -C install/kubernetes
make: Entering directory '/home/kanie2/go/src/github.com/cilium/cilium/install/kubernetes'
: cilium/Chart.yaml
grep -lR -e 'version:' -e 'appVersion:' /home/kanie2/go/src/github.com/cilium/cilium/install/kubernetes/cilium/ \
        | xargs -L 1  sed -i -e 's/''\([vV]ersion:\) ''[0-9]\+\.[0-9]\+\.[0-9]\+.*''/\1 '1.11.90'/g'
sed -i 's;icon:.*;icon: https://cdn.jsdelivr.net/gh/cilium/cilium@master/Documentation/images/logo-solo.svg;' "/home/kanie2/go/src/github.com/cilium/cilium/install/kubernetes/cilium/Chart.yaml"
docker container run --rm --workdir /src/install/kubernetes --volume /home/kanie2/go/src/github.com/cilium/cilium/install/kubernetes/../..:/src --user "1000:1000" "docker.io/bmcustodio/helm-docs:"5ddabba"@sha256:"c4a91daac6bf9c181fac79e09712fcf1f9102fcab1bbacbc733883965f5fd244""
time="2022-06-14T19:41:04Z" level=info msg="Found Chart directories [cilium]"
time="2022-06-14T19:41:04Z" level=info msg="Generating README Documentation for chart /src/install/kubernetes/cilium"
: 
for crd in ciliumclusterwidenetworkpolicies ciliumnetworkpolicies ciliumidentities ciliumendpoints ciliumexternalworkloads ciliumegressgatewaypolicies ciliumnodes ciliumlocalredirectpolicies ciliumenvoyconfigs ciliumclusterwideenvoyconfigs ciliumbgploadbalancerippools ciliumbgppeeringpolicies ciliumendpointslices ciliumegressnatpolicies; do \
        grep -q $crd "/home/kanie2/go/src/github.com/cilium/cilium/install/kubernetes/cilium/Chart.yaml" \
        || (echo -e "$crd not found in "/home/kanie2/go/src/github.com/cilium/cilium/install/kubernetes/cilium/Chart.yaml".\nPlease update the chart to include $crd."; exit 1); \
done
helm lint --with-subcharts --values ./cilium/values.yaml ./cilium
==> Linting ./cilium
[ERROR] templates/: parse error at (cilium/templates/validate.yaml:38): unclosed action

Error: 1 chart(s) linted, 1 chart(s) failed
make: *** [Makefile:53: lint] Error 1
make: Leaving directory '/home/kanie2/go/src/github.com/cilium/cilium/install/kubernetes'
```
I am not sure what to do about that.

Reply:
:thinking: hmm, when I check out your PR and run that command it works. Which version of `helm` do you have installed locally?

Reply:
Worst case you can just skip the lint by doing `make -C install/kubernetes update-versions docs`.

Reply:
I have done the steps and committed the changes. Should I resolve this conversation ?

Reply:
Only some of the commands in that step are run in the container today. I agree that running Helm in a container would be a reasonable way to mitigate this.